src/component/plotly-dash-preview/parse.js

```diff
@@ -41,8 +41,8 @@ function parse (body, req, opts, sendToRenderer) {
   } else if (body.pageSize && isPositiveNumeric(body.pageSize.width) &&
              isPositiveNumeric(body.pageSize.height)) {
     result.browserSize = {
-      width: body.pageSize.width * cst.pixelsInMicron,
-      height: body.pageSize.height * cst.pixelsInMicron
+      width: parseFloat(body.pageSize.width * cst.pixelsInMicron),
```

Comment:
Can you change the commit message to `Ensure that width/height settings are Floats`

Reply:
I think

```js
width: Number(body.pageSize.width) * cst.pixelsInMicron
```

would make things more clear.

Reply:
Not that it matters much here, but `parseFloat` is slow.